--- Services/Mailman.php.orig	2017-09-06 21:15:21.000000000 +0100
+++ Services/Mailman.php	2017-09-06 21:15:33.000000000 +0100
@@ -375,7 +375,7 @@
     {
         $path = '/' . $this->list . '/members/add';
         $query = array('subscribe_or_invite' => (int)$invite,
-                        'send_welcome_msg_to_this_batch' => 0,
+                        'send_welcome_msg_to_this_batch' => 1,
                         'send_notifications_to_list_owner' => 0,
                         'subscribees' => $email,
                         'adminpw' => $this->adminPW);
@@ -608,4 +608,4 @@
         throw new Services_Mailman_Exception('Failed to parse HTML.');
     }
 } //end
-//eof
\ No newline at end of file
+//eof
